.reek

```diff
@@ -28,7 +28,7 @@ NilCheck:
     - PasswordsController#mark_profile_inactive
     - PasswordResetTokenValidator#submit
     - PhoneConfirmationFlow
-    - RegisterUserEmailForm#result
+    - RegisterUserEmailForm#extra_analytics_attributes
```

Comment:
I think it's worth keeping for the cases where a NullObject makes sense to use. In fact, I think this is one of those cases. We already have `AnonymousUser` with `uuid` defined. Let me rework this.